java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -104,7 +106,9 @@ public static Object[] transactionTestResult() {
             1L,
             5L,
             10.5,
-            3L,
+            4L,
+            "OK",
```

Comment:
Import glide.api.BaseClient.OK;
```suggestion
            OK,
```